mkosi/distributions/__init__.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from pathlib import Path
+from typing import TYPE_CHECKING, List
+
+if TYPE_CHECKING:
+    from ..backend import MkosiState
```

Comment:
Hmm, I think the general consensus nowadays is to not use relative imports without a good reason. `from mkosi.backend` is much clearer…

Reply:
This isn't fixed?

Reply:
I'm not sure I follow. The diff is marked as outdated. I fixed that in the commit "Make all imports absolute" and in the version I see here ([b8580b7](https://github.com/systemd/mkosi/pull/1217/commits/b8580b79fc15031cee82efb4e8963aed52e2cb36)), which is the same as I have locally, the import is `mkosi.backend`.

Reply:
Ahh, now it makes sense. In systemd contribution guidelines we always things to be fixed in the commit that introduces them instead of in a later commit, can you do that here as well? Makes things easier to review commit by commit

Reply:
Fair enough, I can rebase that once more, but since this was a thing used all over the codebase, i.e. existing code, I thought it made more sense to keep the relative commits around everywhere and have one point where they are all switched over.